### PR TITLE
Add gitter badge to readme

### PR DESCRIPTION
**Pull request by svozza**
I think it's useful to be able to direct people to our Gitter room as it can be a great way for people new to Sanctuary to get information and advice on issues they might be facing.

### CODE REVIEW

**Comment by davidchambers** [comment]
Good idea!

I'm not sure about the specific graphics:

npm Gitter

It strikes me as a good idea to use a single service for all the badges. This way we can be sure that they'll be consistent. These are the [Shields.io][1] equivalents:

npm Gitter


[1]: https://shields.io/

**Comment by svozza** [comment]
Yeah, I literally took the first ones I found online. I tried to get a CircleCI one but it needs some sort of access token.

**Comment by Shard** [comment]
Here is the the CircleCI Shields.io version that should work:

`[CircleCI](https://circleci.com/gh/Shard/glider-worker) [![codecov]`

**Comment by davidchambers** [comment]
This works too:

CircleCI

**Comment by svozza** [comment]
Thanks guys. :zap:

**Comment by davidchambers** [comment]
Can we have the badges link to the various places?

File: index.js [hunk 1]
```diff
@@ -10,6 +10,11 @@
 
 //. # Sanctuary
 //.
+//.
```
Comment (davidchambers) [inline index.js]: Let's delete this line.

**Comment by svozza** [comment]
:zap:

**Comment by davidchambers** [comment]
Looking good!

We needn't wait for the next release to update __README.md__. Stefano, could you run the following commands to update it?

```console
$ VERSION=0.12.0 make README.md
$ git add -p !$  # press "y" to accept the first change then "q" to reject the rest
$ git checkout !$
```

Please rebase while you're at it. :)

**Comment by svozza** [comment]
I'm not sure if I did that right but I ran the commands, ran my rebase and then pushed.

**Comment by davidchambers** [comment]
Hmm, it looks as though the changes to __README.md__ didn't stick.

**Comment by svozza** [comment]
Ah, I had an error when I pushed. Fixed now.

**Comment by davidchambers** [comment]
:deciduous_tree:

**Pull request merged**